src/bindings/python/docs/contributing.md

```diff
@@ -4,6 +4,11 @@
 *To be added...*
 
 ##### Enviroment
+In case the Python version you have is not supported by OpenVINO, you can refer to 
+
+    openvino/src/bindings/python/docs/python_version_upgrade.md
```

Comment:
should we refer directly to github to make it to be a reference?

Reply:
Good point for user convenience, done.